index.d.ts

```diff
@@ -58,7 +55,49 @@ declare const tempy: {
 	directory(): string;
 
 	/**
-	Get the root temporary directory path. For example: `/private/var/folders/3x/jf5977fn79jbglr7rk0tq4d00000gn/T`.
+	Deletes temporary directories and returns an array of deleted path.
+
```

Comment:
It's not clear to the user when or why they would use this.

I guess the intention is for this to be used for very long-running processes producing a LOT of temp files, otherwise it's better to just rely on the automatic cleanup.